tests/test_permissions.py

```diff
@@ -314,3 +314,30 @@ def test_permissions_denied_head_model_process(factory, client_1, client_2):
     )
     with pytest.raises(substra.exceptions.InvalidRequest):
         client_2.add_composite_traintuple(spec)
+
+
+def test_permission_public_trunk(factory, client):
```

Comment:
It is not really clear what's the purpose of this test. Adding a docstring/comments would be IMO a good idea.